lib/msal-common/src/unifiedCache/serialize/JsonKeys.ts

```diff
@@ -0,0 +1,93 @@
+/*
```

Comment:
This file is weird to me. not sure why we are making classes with static vars.

Reply:
Above comment addresses this!